step_definitions/editTask-steps.js

```diff
@@ -0,0 +1,66 @@
+"use strict";
+
+const navigationLeftPage = require('../page-objects/pages/NavigationLeftPage.js');
+const homeworkListPage = require('../page-objects/pages/HMWRKHomeworkListPage.js');
+const homeworkPage = require('../page-objects/pages/HMWRKHomeworkPage.js');
+const editHomeworkPage = require('../page-objects/pages/HMWRKAddEditHomeworkPage.js');
+
+const { Then } = require('cucumber');
+const { expect } = require('chai');
+
+
+Then(/^he should click the edit-task-button$/, async function(){
```

Comment:
change it to Then(/^.* click the edit-task-button$/, async function(){ , we can use this step for different users, in rest steps below the same